NCEP/docs/source/outputs.rst

```diff
@@ -0,0 +1,13 @@
+######################
+Product
+######################
+
+The GraphCastGFS model runs 4 times a day at 00Z, 06Z, 12Z, and 18Z cycles, which applies to both 13 levels and 37-levels. The 13 pressure levels 
```

Comment:
Can we adjust format:

The GraphCastGFS model runs 4 times a day at 00Z, 06Z, 12Z, and 18Z cycles. The horizontal resolution is on 0.25 degree lat-lon grid. The vertical resolutions are on both 13 levels or 37 pressure levels. 

- The 13 pressure levels include:
50, 100, 150, 200, 250, 300, 400, 500, 600, 700, 850, 925, and 1000 hPa. 

- The 37 pressure levels include:
1, 2, 3, 5, 7, 10, 20, 30, 50, 70, 100, 125, 150, 175, 200, 225, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 825, 850, 875, 900, 925, 950, 975, and 1000 hPa. 

The model output fields are:

- 3D fields on pressure levels:  
 * temperature
 * U and V component of wind
 * geopotential height
 * specific humidity
 * vertical velocity
- 2D surface fields: 
   * 10-m U and V components of wind
   * 2-m temperature
   * mean sea-level pressure
   * 6 hourly total precipitation

The near real-time forecast outputs along with inputs are available on `AWS <https://noaa-nws-graphcastgfs-pds.s3.amazonaws.com/index.html>`_. 

For each cycle, the dataset contains input files to feed into GraphCast found in the directory graphcastgfs.yyyymmdd/hh/input 
and 10-day forecast results for the current cycle found in the following directories:
graphcastgfs.yyyymmdd/hh/forecasts_13_levels 
graphcastgfs.yyyymmdd/hh/forecasts_37_levels, respectively

Reply:
@junwang-noaa I was wondering if this is all the comments from your side? Linlin already merged this, so I asked her to create a very small PR and apply this new comment. Please let us know if this is all?

Reply:
Yes. We may need to double check when we see the doc on readthedocs